python/src/main/python/pygw/store/cassandra/options.py

```diff
@@ -86,3 +103,54 @@ def get_replication_factor(self):
             The number of replicas to use when creating a new keyspace.
         """
         return self._base_options.getReplicationFactor()
+
+    def set_gc_grace_seconds(self, gc_grace_seconds):
```

Comment:
These gc_grace_seconds methods aren't being covered in the test.

Reply:
good catch